Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Devices/GoogleVR.cs

```diff
@@ -1,4 +1,4 @@
-#if UNITY_INPUT_SYSTEM_ENABLE_XR && ENABLE_VR && !DISABLE_BUILTIN_INPUT_SYSTEM_GOOGLEVR && !PACKAGE_DOCS_GENERATION
+#if UNITY_XR_AVAILABLE && !DISABLE_BUILTIN_INPUT_SYSTEM_GOOGLEVR && !PACKAGE_DOCS_GENERATION && !UNITY_FORCE_INPUTSYSTEM_XR_OFF
```

Comment:
These files for specific devices and their defines (e.g.: DISABLE_BUILTIN_INPUT_SYSTEM_GOOGLEVR) all pass the two year LTS mark. These can be safely removed, as all supported XR.SDK packages have device layouts in them.

Reply:
deleted them all. :)